src/main/java/mate/academy/spring/security/PasswordMatchesValidator.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.spring.security;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import mate.academy.spring.dto.UserDto;
+import mate.academy.spring.security.annotations.PasswordMatches;
+
+public class PasswordMatchesValidator
+        implements ConstraintValidator<PasswordMatches, Object> {
```

Comment:
@TatianaPirch Can we use like this?
```suggestion
        implements ConstraintValidator<PasswordMatches, UserDto> {
```